adoc/entities.adoc

```diff
@@ -3,8 +3,8 @@
 :product: {suse} Cloud Application Platform
 :version: 1.5
 :rn-url: https://www.suse.com/releasenotes
-:doc-url: https://www.suse.com/documentation/cloud-application-platform-1
-:deployment-url: https://www.suse.com/documentation/cloud-application-platform-1/book_cap_guides/data/part_cap_deployment.html
+:doc-url: https://documentation.suse.com/suse-cap/1
+:deployment-url: https://documentation.suse.com/suse-cap/1/html/cap-guides/part_cap_deployment.html
```

Comment:
```suggestion
https://documentation.suse.com/suse-cap/1/html/cap-guides/part-cap-deployment.html
```

Reply:
Let me know if this file has been corrected with the correct URLs since this file is used to generate the other files via `make`.

Reply:
The suggested URL (with the hyphens at the end) should be the one used